filebeat/docs/modules/azure.asciidoc

```diff
@@ -47,41 +46,38 @@ image::./images/filebeat-azure-overview.png[]
 ```
 - module: azure
   activitylogs:
-    enabled: true
+    enabled: false
     var:
-      namespace: "obseventhubs.servicebus.windows.net:9093"
-      eventhub: ["insights-operational-logs"]
+      eventhub: "insights-operational-logs"
       consumer_group: "$Default"
       connection_string: ""
+      storage_account: ""
+      storage_account_key: ""
+
    auditlogs:
      enabled: true
```

Comment:
Curious why auditlos and signinlogs are default to be enabled?

Reply:
they are disabled by default, the documentation did not reflect that, resolved the conflicts now.